dockstore-webservice/src/main/java/io/dockstore/webservice/core/OrganizationUser.java

```diff
@@ -39,12 +39,25 @@ public enum Role {
         ADMIN, MAINTAINER, MEMBER
     }
 
+    public enum InvitationStatus { 
+        PENDING, REJECTED, ACCEPTED 
+    }
+
     @Enumerated(EnumType.STRING)
     @ApiModelProperty(value = "The role of the user in the organization", required = true)
     private Role role;
 
+    @Deprecated(since = "1.14.0")
+    @Column(nullable = false)
     @ApiModelProperty(value = "Has the user accepted their membership.", required = true)
-    private boolean accepted = false;
+    @Schema(description = "Has the user accepted their membership.", required = true, deprecated = true)
+    private Boolean accepted;
```

Comment:
Belated thought after this came up in a different meeting with @denis-yuen  -- do we need this at all?

I guess it depends on how the Hibernate schema validator works. Does it allow extra columns that aren't mapped? According to this unverified [StackOverflow question](https://stackoverflow.com/questions/39493603/hibernate-schema-validation-fail-on-additional-columns-tables) it does. The user is complaining that it does, but that would work to our advantage.

If that's true, then I think you can just remove this from the Java code entirely. The 1.14 code will access and only know about the new column, the 1.13 code will access and only know about the old column, and the DB keeps them in sync behind the scenes. Untested theory.

Reply:
I tested this by removing the `accepted` field and the webservice starts with no errors, but I believe this will fail the CircleCI build [here](https://github.com/dockstore/dockstore/blob/develop/.circleci/config.yml#L292) because the migrations are not consistent with the JPA class